islpy/__init__.py

```diff
@@ -107,6 +107,7 @@
 
 error = _isl.error
 stat = _isl.stat
+bool = _isl.bool
```

Comment:
`isl_bool` is already handled in the wrapper generator (and deliberately not exposed as a separate type). `isl_bool_error` is converted to an exception.